src/components/AccountSecurity/MyOrganization.tsx

```diff
@@ -223,6 +223,7 @@ class MyOrganization extends Component<Props, State> {
             <option defaultValue="" disabled hidden aria-labelledby="role" />
             <option value="Admin">Admin</option>
             <option value="Worker">Worker</option>
+            <option value="Client">Client</option>
```

Comment:
Is this related to the Application PDF component? Usually, your PR should just focus on one page/topic to make your changes relevant to each other.

Reply:
No sorry, it is a small change to allow for clients to be enrolled